### PR TITLE
Add harmony to network list

### PR DESCRIPTION
**Pull request by ernestognw**
## Description

Added Harmony to #64 in a separated PR so the other can be released meanwhile we deploy Harmony in prod

### CODE REVIEW

File: packages/base/src/utils/network.ts [hunk 1]
```diff
@@ -93,4 +97,6 @@ const chainIds: { [key in Network]: number } = {
   'arbitrum-rinkeby': 421611,
   celo: 42220,
   alfajores: 44787,
+  ['harmony-s0']: 1666600000,
```
Comment (shahnami) [inline packages/base/src/utils/network.ts]: Curious, why the square brackets?

Comment (shahnami) [reply packages/base/src/utils/network.ts]: Aaaaand it's gone :p

Comment (ernestognw) [reply packages/base/src/utils/network.ts]: A mistake, I already updated 😅

**Review by shahnami — APPROVED** [review]
LGTM!

**Comment by ernestognw** [comment]
Don't merge until https://github.com/OpenZeppelin/defender/pull/2339 is done

**Pull request merged**